src/charts/bar-chart.ts

```diff
@@ -177,19 +184,23 @@ export class BarChart extends StackMixin {
         return safeNumber(x);
     }
 
-    public _renderBars (layer: SVGGElementSelection, layerIndex: number, data): void {
-        const bars: Selection<SVGRectElement, any, SVGGElement, any> = layer.selectAll<SVGRectElement, any>('rect.bar')
+    public _renderBars(layer: SVGGElementSelection, layerIndex: number, data): void {
+        const bars: Selection<SVGRectElement, any, SVGGElement, any> = layer
+            .selectAll<SVGRectElement, any>('rect.bar')
             .data<any>(data.values, d => d.x);
 
-        const enter: Selection<SVGRectElement, any, SVGGElement, any> = bars.enter()
+        const enter: Selection<SVGRectElement, any, SVGGElement, any> = bars
+            .enter()
             .append('rect')
             .attr('class', 'bar')
             .attr('fill', (d, i) => this.getColor(d, i))
             .attr('x', d => this._barXPos(d))
             .attr('y', this.yAxisHeight())
             .attr('height', 0);
 
-        const barsEnterUpdate: Selection<SVGRectElement, unknown, SVGGElement, any> = enter.merge(bars);
+        const barsEnterUpdate: Selection<SVGRectElement, unknown, SVGGElement, any> = enter.merge(
+            bars
+        );
 
```

Comment:
This formatting looks weird. These can be forced with `// prettier-ignore`

Reply:
Is this because the line was too long?

Reply:
Prettier is different than linters. It tries to optimize for human readers, see thier note https://prettier.io/docs/en/options.html#print-width

Reply:
I can't imagine they are doing this for readability. The original line is 106 characters.

Reply:
:smile: I think they would get better over time.